composer.json

```diff
@@ -13,18 +13,20 @@
         "php": ">=5.5",
         "arbiter/arbiter": "^0.1",
         "destrukt/destrukt": "^0.7.1",
+        "equip/adr": "^1.0",
         "filp/whoops": "^1.1",
         "nikic/fast-route": "^0.7",
+        "psr/log": "^1.0",
         "rdlowrey/auryn": "^1.1",
         "relay/relay": "^1.1",
         "relay/middleware": "^0.1",
-        "equip/adr": "^1.0",
         "willdurand/negotiation": "^2.0",
         "zendframework/zend-diactoros": "^1.0.4"
     },
     "require-dev": {
         "josegonzalez/dotenv": "^1.2",
         "league/plates": "^3.1",
+        "monolog/monolog": "^1.17",
         "phpunit/phpunit": "^4.8|^5.0"
     },
     "suggest": {
```

Comment:
I would also add a PR to equip/project that requires monolog there, since we'll need one by default.